### ISSUE TITLE
Edit Type - Template tab displays PHP notice

### ISSUE THREAD

**Issue by michalrames** [FLEXIcontent/flexicontent-cck#534]
Version: Joomla 3.5.1, FLEXIcontent 3.0.14-rc2a and 3.0.13

When creating a new Type, the properties panel throws error on the Template tab.
```
[01-May-2016 11:00:21 UTC] PHP Notice:  Undefined index:  in C:\wamp64\www\joomla\administrator\components\com_flexicontent\views\type\tmpl\default.php on line 172

...

[01-May-2016 11:00:21 UTC] PHP Notice:  Undefined variable: tabSetCnt in C:\wamp64\www\joomla\administrator\components\com_flexicontent\views\type\tmpl\default.php on line 240

...

[01-May-2016 11:00:21 UTC] PHP Notice:  Undefined variable: tabSetCnt in C:\wamp64\www\joomla\administrator\components\com_flexicontent\views\type\tmpl\default.php on line 240

```

**Comment by micker** [comment]
Please flexicontent version and joomla version?

On Sun, May 01, 2016 at 1:03 PM, Michal Rames < notifications@github.com [notifications@github.com] > wrote:
When creating a new Type, the properties panel throws error on the Template tab.

[01-May-2016 11:00:21 UTC] PHP Notice:  Undefined index:  in C:\wamp64\www\joomla\administrator\components\com_flexicontent\views\type\tmpl\default.php on line 172

...

[01-May-2016 11:00:21 UTC] PHP Notice:  Undefined variable: tabSetCnt in C:\wamp64\www\joomla\administrator\components\com_flexicontent\views\type\tmpl\default.php on line 240

...

[01-May-2016 11:00:21 UTC] PHP Notice:  Undefined variable: tabSetCnt in C:\wamp64\www\joomla\administrator\components\com_flexicontent\views\type\tmpl\default.php on line 240

**Comment by michalrames** [comment]
Version: Joomla 3.5.1, FLEXIcontent 3.0.14-rc2a and 3.0.13

Thank you

**Comment by ggppdk** [comment]
Fixed the PHP notices with d1e34ca
